lib/ui/navigator/bottom.dart

```diff
@@ -74,6 +74,8 @@ class _BottomTabBarState extends State<BottomTabBar> {
         ],
         showSelectedLabels: false,
         showUnselectedLabels: false,
+        unselectedFontSize: 0.0,
```

Comment:
Flutter issue https://github.com/flutter/flutter/issues/86545 closed for some reason with no resolution. Testing a work-around fix described in comment https://github.com/flutter/flutter/issues/86545#issuecomment-923699415